bin/Powheg/production/2017/13TeV/DY_MassBinned_MiNNLO_NNPDF31_13TeV/ZJToEE-suggested-nnpdf31-ncalls-doublefsr-q139-m1000to1500-powheg.input

```diff
@@ -0,0 +1,78 @@
+vdecaymode  1     ! Z decay products (default 2): 1 for electronic
+
+bornktmin 0.26
+
+numevts NEVENTS
+ih1   1           ! hadron 1 (1 for protons, -1 for antiprotons)
+ih2   1           ! hadron 2 (1 for protons, -1 for antiprotons)
+ebeam1 6500d0     ! energy of beam 1
+ebeam2 6500d0     ! energy of beam 2
+
+! ALPHAS AND PDF EVOLUTION
+alphas_from_pdf 1 ! (default 0) if 1, uses alphas from PDF evolution tool (e.g. lhapdf or hoppet)
+
+! MINLO SETTINGS
+minlo 1        ! (default 0) if 1, activate minlo
+minnlo 1       ! (default 0) if 1, activate miNNlo
+modlog_p 6     ! (optional, only works if minnlo is set to 1)
+               ! if present, activate modified logs and set the exponent
+inc_delta_terms 0 ! (default 1) if 0, as^3 NNLO delta terms are not included
+sudscalevar 1   ! (default 1) scale variation also in Sudakov form factors in minlo 
+distribute_by_ub 1 ! (default 0) if 1 distribute d3 terms according to kinematics of ub
+distribute_by_ub_AP 1 ! (default 0) if 1 distribute d3 terms according to kinematics of ub
+alphas_cutoff_fact 0.0 ! ! (Higgs: 2.5, DY: 1.8) see setlocalscales.f
+pdf_cutoff_fact 1.1 ! (Higgs: 2.5, DY: 1.8) see setlocalscales.f
+use_NNLOPS_pdfs 1 ! (default 0)
+negative_pdfs_zero 0 ! setting negative PDFs zero
+
+profiledscales 1 ! (default 0) if 1, use the profiled scales with cutoff Q0 and power npow
+largeptscales 0  ! (default 0) if 0, at large pt, use muR=muF~Q in fixed order part
+                 !             if 1, at large pt, use muR=muF=pt in the fixed-order part
+Q0 1.390985      ! (default 2.) Q0 value used in profiled scales (see 2006.04133 for more details)
+                 !              smaller values can be used too.
+npow 1           ! (default 1) power in profiled scales
+
+withdamp 1
+doublefsr 1
+
+! PROCESS PARAMETERS
+# EW input
+
+min_Z_mass 1000d0  ! Lower mass cut-off for Z boson production
+max_Z_mass 1500d0 ! Upper mass cut-off for Z boson production
+
+! To be set only if using LHA pdfs
+lhans1   306000     ! pdf set for hadron 1 (LHA numbering)
+lhans2   306000    ! pdf set for hadron 2 (LHA numbering)
```

Comment:
i have one question on the pdf, since for UL we use 325300 (the positive version of 306000), did you confirm with Kenneth about it, i guess wmass measurement wants those samples initially

Reply:
Dear @menglu21 , thanks for the review! I think the PDF should be synchronized with what was used for M50 sample because the mass-binned samples will be used by combining with M50 sample, but I will ask MiNNLO experts as you said to confirm.

Reply:
I think it's probably correct that we want to keep this synchronised, hopefully the difference between the two is small. I suppose you could include this sample as a weight without regenerating the gridpack.